### PR TITLE
Update the section "What is a booster vaccination..." (vac_booster_basics) in the faq

### LINKED ISSUE #3080: Outdated wait time (3 months) for 1st booster in FAQ
## Where to find the issue

- https://www.coronawarn.app/en/faq/results/#vac_booster_basics "What is a booster vaccination and how does it appear in the CWA?"
- https://www.coronawarn.app/de/faq/results/#vac_booster_basics "Was ist eine Auffrischimpfung und wie wird sie in der CWA dargestellt?"

## Describe the issue

The above FAQ says "A booster vaccination (revaccination) is a vaccination process designed to refresh waning vaccination protection after basic immunization. In most cases, but not always, it takes place according to the recommendations of the authorities after a period of 3 months after the last vaccination procedure. (Example: completion of the vaccination series for basic immunization on 08 Jan 2022, possible booster vaccination on: 08 Apr 2022)".

The German STIKO authority now recommends a 6 month's, not 3 month's wait for the 1st booster. (See [Epidemiologisches
Bulletin 3 2022](https://www.rki.de/DE/Content/Infekt/EpidBull/Archiv/2022/Ausgaben/33_22.pdf?__blob=publicationFile) Stand: 18.08.2022, Page 10, Table 4.) This means that the FAQ is not in alignment with the latest official recommendations.

## Suggestion

Update the FAQ according to the 21st STIKO recommendations, having regard also to the outcome of issue https://github.com/corona-warn-app/dcc-rule-translation/issues/10 questioning whether the app will also be updated to use a recommended wait time of 6 months in the BNR rules.

---
Internal Tracking ID: [EXPOSUREAPP-14105](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14105)

### PR DESCRIPTION
**Pull request by brianebeling**
Updates both the faq in German and English with new texts.

When merging, it'd be best to squash, because of the small commits.

Fixes #3080 
Partly addresses #2345 by removing the section 

Screenshots:
en
de

---
Internal Tracking ID: [EXPOSUREAPP-14105](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14105)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
The section about the Impf-Guide could be improved:

1. Reduce the text so that "Impf-Guide" is only mentioned once instead of twice
2. Add a period at the end of the sentence
3. In the English text make a note that the "Impf-Guide" is in German

**Comment by Ein-Tim** [comment]
Also, please consider removing the information that boosters where recommended after 3 months in the past. It is not relevant anymore and makes the text unnecessary complex.

File: src/data/faq_de.json [hunk 1]
```diff
@@ -871,7 +871,7 @@
...
...in die Corona-Warn-App einlesen können. <b>Wichtig</b>: Bitte achten Sie schon direkt beim Ausstellen des Zertifikats darauf, dass die persönlichen Daten (Titel, Name, Vornamen, und Geburtsdatum) passend zu den bereits existierenden Impfzertifikaten (und auch ggfs. Genesenenzertifikaten) ausgewiesen werden. Nur so kann die Corona-Warn-App das neue Zertifikat zu den anderen Zertifikaten (Impfungen bzw. Genesung) zuordnen (Beispiel: gleicher akademischer Titel oder Angabe des vollständigen Doppelnamens). Alle Zertifikate einer Person sollten in der Corona-Warn-App gruppiert angezeigt werden, siehe Abbildung.</p><p>Auffrischimpfungen bieten einen sofortigen Impfschutz.</p><p>Basierend auf den bisherigen Impfvorgängen, wird das erste Zertifikat einer Auffrischimpfung mit '4 von 4' (bei vorheriger Grundimmunisierung mit 3 Dosen) ausgestellt.<p><p>Die Kennzeichnung '2 von 1' im Zertifikat einer Impfung zeigt an, dass es sich um den ersten Impfvorgang nach einer Genesung handelt.</p><p>Siehe FAQ-Artikel: <a href='#vac_booster_jj' target='_blank'>Warum erscheint in der Corona-Warn-App eine 14-tägige Wartezeit bis zum vollen Impfschutz nach einer Auffrischimpfung?</a></p></div></div></div>"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:874]: For the new paragraph suggest:

<p>Unter <a href='https://www.zusammengegencorona.de/impfen/#id-862101356'>Impf-Guide</a> vom Bundesministerium für Gesundheit (BMG) können Sie Ihre persönlichen Impfempfehlung von der Ständigen Impfkommission (STIKO) erstellen.</p>

File: src/data/faq.json [hunk 2]
```diff
@@ -871,7 +871,7 @@
...
...ccination recommendation from the Standing Committee on Vaccination (STIKO)</p><p>Each vaccination can be attested by a digital COVID certificate, which you can scan into the Corona-Warn-App. <b>Important</b>: please ensure that the personal data (surname, first name and date of birth) are shown to match the existing vaccination certificates (and also recovery certificates, if applicable). Only then will it be possible to assign the new certificate to the other certificates (vaccinations or recovery). (Example: same academic title or indication of the full double name). All certificates of a person should be displayed grouped in the Corona-Warn-App, see figure. </p><p>Booster shots provide immediate vaccine protection.</p><p>Based on the previous vaccination series, the booster certificate will be documented '4 of 4' (in case of previous basic immunization with 3 doses).</p><p>The '2 of 1' label for a vaccination indicates that it is the first vaccination after a COVID-19 disease recovery.</p><p> See also: <a href='#vac_booster_jj' target='_blank'>Why does a 14-day wait for full vaccine protection after a booster vaccination appear in the Corona-Warn-App?</a></p></div></div></div>"
```
Comment (MikeMcC399) [inline src/data/faq.json:874]: <p>An interactive German-language vaccination guide from the Federal Ministry of Health (BMG) is available under <a href='https://www.zusammengegencorona.de/impfen/#id-862101356'>Impf-Guide</a> which provides a personal vaccination recommendation according to the rules of the Standing Committee on Vaccination (STIKO).</p>

**Comment by MikeMcC399** [comment]
I added some concrete suggestions for changed wording.

File: src/data/faq_de.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq_de.json:874]: @Ein-Tim 
Thanks! I changed the suggestion.

Comment (brianebeling) [reply src/data/faq_de.json:874]: Thank you both, included it!

**Comment by brianebeling** [comment]
> Also, please consider removing the information that boosters where recommended after 3 months in the past. It is not relevant anymore and makes the text unnecessary complex.

I think adding the 3 months, is more beneficial than removing it. Many are confused by sudden changes and this should help ease it out a bit. What do you think?

**Review by MikeMcC399 — APPROVED** [review]

**Comment by Ein-Tim** [comment]
@brianebeling 

Okay, agreed!

**Review by Ein-Tim — APPROVED** [review]
LGTM!

**Review by GisoSchroederSAP — APPROVED** [review]
Looks consistent and correct; approved

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim Many thanks for suggestions and reviewing! Enjoy the weekend!

**Comment by MikeMcC399** [comment]
I missed that fact that the new link does not abide by the conventions specified in https://github.com/corona-warn-app/cwa-website/blob/master/docs/CHANGE_WEBSITE_CONTENT.md#links-on-faq-pages:

## Links on FAQ pages

The following conventions should be followed for links in the FAQ sources [/src/data/faq.json](https://github.com/corona-warn-app/cwa-website/blob/master/src/data/faq.json) and [/src/data/faq_de.json](https://github.com/corona-warn-app/cwa-website/blob/master/src/data/faq_de.json):

- All internal and external links on the FAQ pages should use the HTML hyperlink attribute `target='_blank'` to open in a new frame. This provides a consistent user experience and works around an issue with the browser back arrow button in the FAQ Glossary.
- External links should use the HTML hyperlink attribute `rel='noopener noreferrer'` to avoid a cross-domain security issue in older browsers when the link also contains `target='_blank'`.
- Internal links should **not** contain the attribute `rel='noopener noreferrer'`. There is no security issue for internal links.

**Comment by dsarkar** [comment]
@MikeMcC399 Alrighty, we will fix that on Monday. Thanks for pointing that out!

**Pull request merged**